Dockerfile

```diff
@@ -24,4 +24,4 @@ COPY . /kolibri
 
 VOLUME /kolibridist/  # for mounting the whl files into other docker containers
 # add buildkite pipeline specific installation here:
-CMD cd /kolibri && pip install -r requirements/dev.txt && pip install -r requirements/build.txt && yarn install && make dist pex && cp /kolibri/dist/* /kolibridist/
+CMD cd /kolibri && pip install -r requirements/dev.txt && pip install -r requirements/build.txt && pip install -e . && yarn install && make dist pex && cp /kolibri/dist/* /kolibridist/
```

Comment:
@aronasorman @lyw07 do you have any concerns about doing this here? It is required in order to access some internal Kolibri modules in the build tools.

Reply:
I'm not really sure about this... But I get an error running the pex file. It's
```
WebpackError at /setup_wizard/
Webpack build file missing, front-end assets cannot be loaded
```

Reply:
Looks like it is not building any plugins, only the core bundle!